src/mapproxyUtils/mapproxySeed.ts

```diff
@@ -106,24 +125,80 @@ export class MapproxySeed {
     return validSeedDateFormatted;
   }
 
+  //TODO - should be integrated to update job status-progress mechanism
+  //     - calculate dynamically the actual percentage
+  //     - send update percentage to total percentage on job-task tables
+
+  /**
+   * Not implemented - should calculate task progress dynamically and update job-manager progress percentage
+   * @param {string} seedLogStr current batch stdout
+   * @returns {void}
+   */
+
+  public seedProgressFunc(seedLogStr: string): void {
+    this.logger.debug(seedLogStr); // print all mapproxy-seed stdout
+    if (seedLogStr.match(/- ERROR -/g) && !isErroredCmd) {
+      // substr that detect seeding process error on mapproxy-seed util
+      // task will fail on case of seeding logic error (for example redis connection)
+      isErroredCmd = true;
+      const errMsg = seedLogStr.split('- ERROR -')[1];
+      this.logger.error(errMsg);
+      this.abortController.abort(errMsg);
+      console.log('0000');
```

Comment:
console? - make sure no other console logs exists and change to "logger" when needed